### DISCUSSION TITLE
[Bug] Category images aren't properly rendered in the dashboard

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#366] [Help] [answered]
Category images take up a lot of space in the dashboard, unlike product images. This is how they look:

![sense nom (1)](https://github.com/swellstores/community/assets/37710726/b9dc5e73-ed7d-405c-a872-f7b7783c597d)

I was hoping someone could correct this. Thanks!

**Comment by SusannaSwell** [comment, marked as answer]
hello @marnauortega  I was able to reproduce the issue on our end, let me file it in our internal system. 
Once we have an update, I will get back to you.

**Comment by steven-k1** [comment]
@marnauortega A fix has been made for this issue.

**Reply by marnauortega** [reply]
That's wonderful. Thanks, @steven-k1 !
